staticaddr/address/manager.go

```diff
@@ -46,30 +51,20 @@ type ManagerConfig struct {
 type Manager struct {
 	cfg *ManagerConfig
 
-	initChan chan struct{}
-
 	sync.Mutex
 }
 
-// NewAddressManager creates a new address manager.
-func NewAddressManager(cfg *ManagerConfig) *Manager {
+// NewManager creates a new address manager.
+func NewManager(cfg *ManagerConfig) *Manager {
+	log = staticaddr.GetLogger()
```

Comment:
nit: alternatively, you could do this in the package's `init()` function too, so it's executed only once at the static initialization stage.

Reply:
good idea, obvious in hindsight.